README.md

```diff
@@ -48,6 +48,11 @@ nile init
 pip install openzeppelin-cairo-contracts
 ```
 
+> ⚠️ Warning! ⚠️
+<br>
+Installing directly through GitHub may contain incomplete or breaking implementations.
+While we aim not to introduce such changes, we still strongly recommend installing through [official releases](https://github.com/OpenZeppelin/cairo-contracts/releases/).
```

Comment:
```suggestion
Installing directly the `main` branch may contain incomplete or breaking implementations, download [official releases](https://github.com/OpenZeppelin/cairo-contracts/releases/) only.
```

this wording might be more accurate. installing through git, not just github, should also be discouraged